docs/repositories.md

```diff
@@ -7,11 +7,13 @@ You can download a copy of the repo ("clone"), create a development "branch", ma
 
 GitHub is the means by which code in the repository is passed to the server to be run against the OpenSAFELY database &mdash; it is the only entry point between the secure server and the outside world. GitHub is also the means by which approved disclosure-safe outputs are released from the secure server to researchers.
 
-## Repository access
+## Repository visibility
 
-During the development stage of a project, a repository is typically private, so that only core OpenSAFELY members and project-specific collaborators will have access. See the [permissions page](permissions.md) for more details.
+In accordance with the [Principles of OpenSAFELY](https://www.opensafely.org/about/#transparency-and-public-logs), we recommend that repositories are made public from the start. A public GitHub repository is visible by anyone on the internet. However, only specified people will have the ability to make changes to it.
 
-Once a project is complete, or retired, the repository will be made public. 
+During the development stage of a project, a repository may be kept private, so that only members of the OpenSAFELY GitHub organisation are able view it.
+
+A repository will need to be made public if it forms part of [a publication](https://www.opensafely.org/policies-for-researchers/#acknowledgment-and-data-sharing--publication-policy). We have a [guide to publishing repositories](publishing-repo.md) that we advise you to read. For unfinished or abandoned research, we strongly encourage that repositories are made public within 12 months after the code is first run against an OpenSAFELY database. You can request that a private repository is made public at any time by contacting [tech support](how-to-get-help.md).
```

Comment:
Is there any need for distinction here? Will we not simply make any repo public 12 months after the first time code is run?

Reply:
It may be made public sooner if for some reason the work is published earlier than 12 months. People may also choose to make it public before then.

Technically, we're asking people to agree to make it public rather than forcing it to be public too.

Reply:
This seems to be more permissive than what's written [here](https://www.opensafely.org/about/#transparency-and-public-logs).

I'm not sure exactly what is in any agreements signed in an application, but the below suggests this is made clear.

> OpenSAFELY requires all researchers to archive and publish their analytic code: this is the only way they are allowed to run code against real data

>

Reply:
Links need fixing, sorry 

Also just to check, they do have to _ask an admin_ to make it public, right? Amir has been saying to people they can do it themselves, following approval. If he needs correctly, please let him know in Slack. If not, the above needs tweaking